roles/rsyslog/tasks/outputs/remote_files/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Create files output configuration files in /etc/rsyslog.d
+  vars:
+    __rsyslog_packages: "{{ __rsyslog_remote_files_output_packages }}"
+    __rsyslog_rules: [ { "name": "output-files-{{ remote_files_item.name }}", "type": "output",
+                         "sections": [ { "options": "{{ lookup('template', 'output_remote_files.j2') }}" } ] } ]
```

Comment:
I suggest cleaning this up to put it into a more yaml/ansible friendly structure